flixel/graphics/frames/FlxFramesCollection.hx

```diff
@@ -62,6 +62,20 @@ class FlxFramesCollection implements IFlxDestroyable
 			parent.addFrameCollection(this);
 	}
 
+	/**
+	 * Merges multiple `FlxFramesCollection` and returns the result of the concatenation.
+	 * @param	FrameList	An array that contains various instances of `FlxFramesCollection` (they must contain a spritesheet, see the API for how to initialize using a spritesheet)
+	**/
+	public static function mergeSheets(sheets:Array<FlxFramesCollection>):FlxFramesCollection
+	{
+		var tex:FlxFramesCollection = null;
+		for (i in 0...sheets.length)
+		{
+			tex = tex.frames.concat(i.frames);
```

Comment:
have you tested this? wouldn't this throw a null object ref?

Reply:
yea i tested
for me works fine
but prob ur right i might edit it

Reply:
> have you tested this? wouldn't this throw a null object ref? tex is null

btw i removed that null thing + added some more stuff

Reply:
text is still null, in fact I you just added a compile warning

Reply:
-.- i suck at coding but tf am i supposed to make an instance from nowhere?? Xd

Reply:
i might create a new instance using new()

Reply:
That's one way to do this. more importantly, I just want you to test your changes before making a PR

Reply:
oops sorry just i dont really do PR i just do code locally

btw bout the issue

im gonna do the idea i had